frontend/public/components/namespace.jsx

```diff
@@ -5,12 +5,13 @@ import { Tooltip } from './utils/tooltip';
 import { Link } from 'react-router-dom';
 import * as fuzzy from 'fuzzysearch';
 
-import { NamespaceModel, ProjectModel, SecretModel } from '../models';
+import { NamespaceModel, ProjectModel, RoleBindingModel, SecretModel } from '../models';
```

Comment:
nit: break long import lists into multiple lines and sort.

Reply:
Also, I don't know if we want to make this a convention, but I've been sorting multi-line imports last in the overall list of imports. I've also been breaking the imports into three separately sorted sections: external imports, ancestor imports, and sibling imports. Since some components are now nested further into the file structure, the three sections make it easier to quickly see where dependencies are coming from.